modules/virt-creating-vm-cli.adoc

```diff
@@ -1,16 +1,27 @@
 // Module included in the following assemblies:
 //
-// * virt/virtual_machines/creating_vms_rh/virt-creating-vms-from-cli.adoc
+// * virt/creating_vms_advanced/creating_vms_cli/virt-creating-vms-from-cli.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="virt-creating-vm-cli_{context}"]
 = Creating a VM from a VirtualMachine manifest
 
-You can create a virtual machine (VM) from a `VirtualMachine` manifest.
+You can create a virtual machine (VM) from a `VirtualMachine` manifest. To simplify the creation of these manifests, you can use the `virtctl` command-line tool.
+
+.Prerequisites
+
+* You must have the `virtctl` command-line tool installed.
 
 .Procedure
 
-. Edit the `VirtualMachine` manifest for your VM. The following example configures a {op-system-base-full} VM:
+. Create a `VirtualMachine` manifest for your VM and save it as a YAML file. For example, to create a minimal {op-system-base-full} VM, run the following command:
```

Comment:
I find this a bit confusing. In the first sentence, you have me creating a YAML manifest. Then, in the second, you say I can create a VM using the command. Looking up the command on line, I understand what is going on. I don't want to hold up the merge over it; but, maybe for a future change:

 ```suggestion
. Create a `VirtualMachine` manifest for your VM by using the `virtctl crate vm` command and save it as a YAML file. For example, to create a manifest for a minimal {op-system-base-full} VM, run the following command:
```